### PR TITLE
feat(pack): add gtr default value for migration offer

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #UXCT-590
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~Only FR translations have been updated~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description

add gtr default value for migration offer

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js [hunk 1]
```diff
@@ -252,8 +252,27 @@ export default class TelecomPackMigrationOffersCtrl {
   }
 
   setGtrOptions() {
+    // Check if current offer has a GRT option set
+    this.isGrt = !!Object.keys(this.process.pack.options).find((value) =>
+      value.includes('grt'),
```
Comment (qpavy) [inline packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js:257]: Use a constant file to define this value

File: packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js [hunk 2]
```diff
@@ -252,8 +252,27 @@ export default class TelecomPackMigrationOffersCtrl {
...
+    );
+
+    // Retrieve current offer technology: xdsl or fiber
+    this.currentOfferTechno = this.process.pack.offerDescription
+      .toLowerCase()
+      .includes('xdsl')
```
Comment (qpavy) [inline packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js:263]: Use a constant file to define this value

File: packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js [hunk 3]
```diff
@@ -266,16 +285,28 @@ export default class TelecomPackMigrationOffersCtrl {
             ),
           }));
 
-        if (gtr.length > 0) {
+        if (this.isGrt && this.currentOfferTechno === offerTechno) {
+          this.grtLabel = `telecom_pack_migration_${gtr[0].name}`;
+          this.grtLabelPrice = gtr[0].optionalPrice.text;
+
+          // Initialize GTR selected to first value
+          set(offer, 'gtrSelected', gtr[0].name);
```
Comment (qpavy) [inline packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js:293]:
```suggestion
          offer.gtrSelected = gtr[0].name
```

Comment (Steffy29) [reply packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js:293]: No we can't do that, we are into a map to add some elements need to complete data to display into offers list.

**Review by qpavy — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**